src/biomappings/resources/mappings.tsv

```diff
@@ -2822,6 +2822,671 @@ doid	DOID:8850	salivary gland cancer	skos:exactMatch	mesh	D012468	Salivary Gland
 doid	DOID:9335	scotoma	skos:exactMatch	mesh	D012607	Scotoma	manually_reviewed	orcid:0000-0001-9439-5346
 doid	DOID:9383	iridocyclitis	skos:exactMatch	mesh	D015863	Iridocyclitis	manually_reviewed	orcid:0000-0001-9439-5346
 doid	DOID:9675	pulmonary emphysema	skos:exactMatch	mesh	D011656	Pulmonary Emphysema	manually_reviewed	orcid:0000-0001-9439-5346
+drugbank	DB00001	Lepirudin	skos:exactMatch	drugcentral	2995	lepirudin	manually_reviewed	orcid:0000-0001-5705-7831
```

Comment:
The above is definitely on the DrugCentral website, it's odd that it didn't get filtered out - same for a couple of others that I randomly checked below.

Reply:
I also ran into this for most (if not all?) that I checked. I am not sure what's going on/why they aren't in the database's xrefs table but they appear in the site